### ISSUE TITLE
Allow all card styles to be added to remaining list type blocks

### ISSUE THREAD

**Issue by pyrello** [uiowa/uiowa#2351]
# Story

As a webmaster, I want the ability to add recent card style options to my list blocks and single event blocks.

# Proposed Solution

This was recently done in https://github.com/uiowa/uiowa/pull/2644 and https://github.com/uiowa/uiowa/pull/2683 that set a foundation for this, but that must be extended to the other types as well as adapted based on how they are rendered (view, external events data, etc.).

- [x] Update the card styles configuration to extend to the other block types (articles, people, events, event)
- [x] Update layout_builder_custom to have the same show/hide options functionality.
- [x] Set the defaults for each block type by what is desired going forward.
  - [x] RSS and Article  = white, left align, images are right align wide small
  - [x] Event, Events, Person = white, left align, images are left align small circle
- [x] Write at least one update hook to match the defaults styles already out there in the wild so that the rollout is transparent for existing blocks.
- [x] Make sure the image view mode switching and caching is working based on the render of the current block.
- [x] Update block templates to remove hardcoded classes and instead take classes set by the block settings.
- [ ] Remove, update any uids_base specific styles.
- [ ] Check against v2 uids_base sites

A lot of this work can be referenced in those two previous PRs.

<details>
<summary>Original issue summary</summary>
# Story

As a site owner I want to be able to apply all the card style options to list blocks that utilize the card template. 

Related to  

# Proposed solution

After #2514 is completed, add these options to the remaining list blocks: articles, events, people. Additional consideration should be given to whether there is any possibility of this work affect V2 sites and whether testing will need to be done.
</details>

**Comment by pyrello** [comment]
Brought over from #2306

**Comment by bspeare** [comment]
We need this option for:

- Aggregator
- Events
- Event
- People
- Articles 

Are we able to use the existing layout builder styles on the custom block views listed above?  Or will it have to be recreated for each list type block?  Is there a list type not listed above that we are missing?

<img width="464" alt="Screen Shot 2020-11-06 at 9 27 10 AM" src="https://user-images.githubusercontent.com/1036433/98383656-6622f800-2012-11eb-8ddb-2c5549d585d6.png">

**Comment by bspeare** [comment]
This functionality is needed on the admissions site so that they can adjust the display configuration of their blog feeds, counselors, and student profiles.

**Comment by joewhitsitt** [comment]
@bspeare do we want to take this opportunity to change the default card styles for new blocks of each type or keep them the same as existing blocks out there?

- event
- article
- events
- person

**Comment by bspeare** [comment]
@joewhitsitt That's a good question that might be helpful to get broader input on.  How soon do we need to decide?

**Comment by joewhitsitt** [comment]
Sounds like work will start today, but defaults for new blocks can be swapped out right before merge.

**Comment by bspeare** [comment]
@joewhitsitt That could be a good question to ask at the admissions design meeting today at 11.  Can you drop in for the first 10 minutes to ask this?

**Comment by joewhitsitt** [comment]
Updated the IS to include feedback about defaults going forward

**Comment by joewhitsitt** [comment]
Established a draft pr for this and prepared the event template for the new options. https://github.com/uiowa/uiowa/pull/2730

**Comment by joewhitsitt** [comment]
This got complicated because I think v2 is using the same template for these... I didn't encounter this with aggregator because we didn't extend the functionality to v2.

**Comment by joewhitsitt** [comment]
Closing in favor of new issues to create ListBlock replacements that can implement the card styles as part of the deliverable as well as a clean up task for unused styles that remain in uids_base.

https://github.com/uiowa/uiowa/issues/3489
https://github.com/uiowa/uiowa/issues/3501
https://github.com/uiowa/uiowa/issues/3502
